versioning/revisions.rst

```diff
@@ -76,6 +76,24 @@ Take into account that it changes the default Conan behavior. e.g:
     - If you generate and upload N binary packages for a recipe with a given revision, then if you modify the recipe, and thus the recipe
       revision, you need to build and upload N new binaries matching that new recipe revision.
 
+GIT and Line Endings on Windows
+-------------------------------
+
+.. warning::
+
+  **Problem**
+
+  Git will (by default) checkout files in Windows systems using CRLF line endings, effectively producing different files. As files are different, the Conan revisions will be different from the revisions computed in other platforms such as Linux, resulting in missing the respective binaries in the other revision. 
```

Comment:
is it really true? as I understand ([thread](https://stackoverflow.com/questions/39408793/git-core-autocrlf-line-ending-default-setting/48230871)), the default value of `core.autocrlf` is `false`, it means git will not modify the line endings of the file, which should effectively result into the same revision for all systems.

Reply:
Because the windows installer GUI default selection is true, it's ends up getting set to "true" incidentally for some major number of people without them realizing it. Also, silent install ends up the same, and lots of people use silent install (`choco install git` etc) .

>- the default value is "false"
>- Windows installer let you choose the desired behavior but by default (if you install without changing proposed settings) it sets it to "true". This is not the software default, the installer sets the core.autocrlf system setting.